packages/notification-service/tsconfig.eslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "src/**/*.ts",
+    "test/**/*.ts"
```

Comment:
Nothing to do here, but looks like we should move these tests to be directly in `src`. This way we could avoid the extra tsconfig for this.